taca/utils/bioinfo_tab.py

```diff
@@ -143,7 +158,7 @@ def get_ss_projects(run_dir):
                 FCID_samplesheet_origin = os.path.join(run_dir,'SampleSheet.csv')
                 if not os.path.exists(FCID_samplesheet_origin):
                     logger.warn("Cannot locate the samplesheet for run {}".format(run_dir))
-                    return ['UNKNOWN']
+                    return []
```

Comment:
are you on seqmaster ? We'll barely see any difference.

On Tue, Dec 1, 2015 at 11:36 AM, Francesco <notifications@github.com> wrote:

> In taca/utils/bioinfo_tab.py
> <https://github.com/SciLifeLab/TACA/pull/149#discussion_r46262945>:
>
> > @@ -143,7 +158,7 @@ def get_ss_projects(run_dir):
> >                  FCID_samplesheet_origin = os.path.join(run_dir,'SampleSheet.csv')
> >                  if not os.path.exists(FCID_samplesheet_origin):
> >                      logger.warn("Cannot locate the samplesheet for run {}".format(run_dir))
> > -                    return ['UNKNOWN']
> > +                    return []
>
> yep sound reasonable. Let us keep in mind in case we get tons of mails :-D
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SciLifeLab/TACA/pull/149/files#r46262945>.
>